analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ConditionEvaluatesToConstant.CSharp7.cs

```diff
@@ -294,7 +294,7 @@ public void GoGoGo()
         {
             var tmp = 0;
             var flag = true;
-            while (flag)            // Noncompliant
+            while (flag)            // Compliant
```

Comment:
This was an FP, but wasn't marked as one earlier (see [this issue](https://github.com/SonarSource/sonar-dotnet/issues/3239)).